compiler/luci/pass/src/ResolveBuiltinOpAddPass.cpp

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright (c) 2024 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "luci/Pass/ResolveBuiltinOpAddPass.h"
+
+#include <luci/IR/CircleNodes.h>
+#include <luci/IR/AttrFusedActFunc.h>
+#include <luci/Profile/CircleNodeOrigin.h>
+
+#include <flatbuffers/flexbuffers.h>
+namespace
+{
+
+/// @brief Returns the index of BroadcastTo node among cop's inputs.
+// NOTE This function assumes there is only one BroadcastTo node among its inputs.
```

Comment:
```suggestion
/// @note This function assumes there is only one BroadcastTo node among its inputs.
```

Reply:
> This function assumes there is only one BroadcastTo node among its inputs.

There should be some checks for this assumption.
@mhs4670go , I see `ResolveCustomOpAddPass.cpp` also does not have this check. Can you please revise `ResolveCustomOpAddPass.cpp` ?

Reply:
@Seunghui98 Please refer to [this PR](https://github.com/Samsung/ONE/pull/12480).